plancton/__init__.py

```diff
@@ -469,9 +469,12 @@ def main_loop(self):
       self._influxdb_setup()
     running = self._count_containers()
     self.logctl.debug("CPU used: %.2f%%, available: %.2f%%" % (self.efficiency, self.idle))
-    self.streamer(series="measurement",
+    self.streamer(series="host",
                   tags={ "hostname": self._hostname },
-                  fields={ "cpu_eff": self.efficiency, "containers": running })
+                  fields={ "cpu_eff": self.efficiency })
+    self.streamer(series="daemon",
+                  tags={ "hostname": self._hostname },
+                  fields={ "containers": running })
```

Comment:
You have a point. Would it be possible then to send it _once_, at lines 456-458, along with the rest of the information?